ansible/README.md

```diff
@@ -1,33 +1,108 @@
+   Customise log-rotation
+ - openmicroscopy.lvm-partition
+   ([GitHub](https://github.com/openmicroscopy/ansible-role-lvm-partition)/[Galaxy](https://galaxy.ansible.com/openmicroscopy/lvm-partition/)):
+   Create a formatted LVM volume
+ - openmicroscopy.munin
+   ([GitHub](https://github.com/openmicroscopy/ansible-role-munin)/[Galaxy](https://galaxy.ansible.com/openmicroscopy/munin/)):
+   Setup a Munin monitoring server
+ - openmicroscopy.nfs-mount
+   ([GitHub](https://github.com/openmicroscopy/ansible-role-nfs-mount)/[Galaxy](https://galaxy.ansible.com/openmicroscopy/nfs-mount/)):
+   Manage NFS4 mounts
+ - openmicroscopy.nginx
+   ([GitHub](https://github.com/openmicroscopy/ansible-role-nginx)/[Galaxy](https://galaxy.ansible.com/openmicroscopy/nginx/)):
+   Install upstream nginx
+ - openmicroscopy.postgresql
+   ([GitHub](https://github.com/openmicroscopy/ansible-role-nginx)/[Galaxy](https://galaxy.ansible.com/openmicroscopy/nginx/)):
+   Install upstream PostgreSQL
+ - openmicroscopy.redis
+   ([GitHub](https://github.com/openmicroscopy/ansible-role-nginx)/[Galaxy](https://galaxy.ansible.com/openmicroscopy/nginx/)):
+   Install redis
+ - openmicroscopy.sudoers
+   ([GitHub](https://github.com/openmicroscopy/ansible-role-nginx)/[Galaxy](https://galaxy.ansible.com/openmicroscopy/nginx/)):
+   Configure sudoers
+
+Other core roles can be found under the [openmicroscopy](https://galaxy.ansible.com/openmicroscopy/) account on Galaxy.
+
+
+### Development roles ###
+
+Roles which are not available on Galaxy should be considered in development. For exapmle:
+
+ - hosts-populate
+   ([GitHub](https://github.com/openmicroscopy/ansible-role-hosts-populate)):
+   Populates /etc/hosts with static addresses for a list of hostgroups
+ - omero-user
+   ([GitHub](https://github.com/openmicroscopy/ansible-role-omero-user)):
+   Create OMERO user accounts and groups
+ - omero-web-apps
+   ([GitHub](https://github.com/openmicroscopy/ansible-role-omero-web-apps)):
+   Install OMERO.web plugins
+ - reboot-server
+   ([GitHub](https://github.com/openmicroscopy/ansible-role-reboot-server)):
+   Reboot a server, optionally wait for it to return.
+
+### Community roles ###
+
+The OME community also provides roles. For example:
+
+ - hajaalin.truststore
+   ([GitHub](https://github.com/hajaalin/ansible-role-truststore)/([Galaxy](https://galaxy.ansible.com/hajaalin/truststore/)):
+   Install JKS TrustStore
+
+
+Further reading
+---------------
+
+- [Getting started](../docs/ansible/installation.md): Installation docs and initial examples.
+- [Ansible](../docs/ansible/ansible.md): Overview of Ansible, the configuration management system.
+- [Example workflows](../docs/ansible/example_workflows.md): Examples of provisioning new hosts, and running Ansible playbooks.
+- [Contributing](../docs/ansible/contributing.md): Suggestions on submitting modifications and extensions to the OME ansible roles and playbooks.
 
 
 OME OpenStack Ansible
```

Comment:
`README-os.md` was removed in https://github.com/openmicroscopy/infrastructure/pull/103
`README-os-idr.md` was removed in https://github.com/openmicroscopy/infrastructure/pull/154